codalab/worker/worker.py

```diff
@@ -622,6 +626,16 @@ def initialize_run(self, bundle, resources):
                 bundle=bundle,
                 bundle_path=os.path.realpath(bundle_path),
                 bundle_dir_wait_num_tries=Worker.BUNDLE_DIR_WAIT_NUM_TRIES,
+                bundle_profile_stats={
+                    RunStage.PREPARING: self.init_stage_stats(),
```

Comment:
Let's do this for all the keys, so that the key will actually be equal to the value of `RunStage.PREPARING` rather than the actual literal string `"RunStage.PREPARING"`
```suggestion
                    [RunStage.PREPARING]: self.init_stage_stats(),
```

Reply:
@adiprerepa I believe you didn't address this yet -- could you take a look?

Reply:
is it not addressed? just putting brackets around each `RunStage.*`?

Reply:
Looks like you addressed it here, but there aren't brackets here: https://github.com/codalab/codalab-worksheets/pull/3370/files#diff-2faf6aac04322764cf1ab1ca223e08f26783c1a7aa03b125a5468c246c1916a9R272-R278

Moreover, by adding brackets, that means that this code should also change: https://github.com/codalab/codalab-worksheets/pull/3370/files#diff-66dcf239a8feb8b8d4031040dca05ead4e0d27ec4f580e7987e858028c46a82fR948-R961

Reply:
@adiprerepa , I think you still forgot to add brackets to `tests/unit/server/bundle_manager/__init__.py`.

That also might be why the tests are failing. Could you address this issue and ensure CI is passing before requesting another review?

Reply:
Doesn't adding the brackets make the element a list? `bundle_profile_stats` is a dictionary, so the way it is right now, the value of `RunStage.PREPARING` (`'RUN_STAGE.PREPARING'`) is the key of the element.

Reply:
We discussed this over slack and decided to remove the brackets.